Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -555,6 +555,45 @@ def prepContinents(fnm):
     return poly
 
 
+def apply_proj_parameters(pd, projection, xm, xM, ym, yM):
+    pname = projDict.get(projection._type, projection.type)
+    projName = pname
+    pd.SetName(projName)
+    if projection.type == 'aeqd':
+        # this is a temporary branch to keep the same
```

Comment:
I agree but since it was here before I left it in. I might mean something to someone...